content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md

```diff
@@ -101,7 +103,7 @@ Kubernetes卷插件系统已经提供了功能强大的抽象用于自动配置
 The Kubernetes volume plugin system already provides a powerful abstraction that automates the provisioning, attaching, and mounting of block and file storage.
 -->
 
-支持所有这些特性是Kubernets负载可移植的目标：Kubernetes旨在分布式系统应用和底层集群之间创建一个抽象层,使得应用可以不感知其运行集群的具体信息并且部署也不需特定集群的知识。
+支持所有这些特性是Kubernetes负载可移植的目标：Kubernetes旨在分布式系统应用和底层集群之间创建一个抽象层,使得应用可以不感知其运行集群的具体信息并且部署也不需特定集群的知识。
```

Comment:
```suggestion
支持所有这些特性是 Kubernetes 负载可移植的目标：Kubernetes 旨在分布式系统应用和底层集群之间创建一个抽象层,使得应用可以不感知其运行集群的具体信息并且部署也不需特定集群的知识。
```

Reply:
I know there should be a space between English and Mandarin words, but this PR is limited to fixing typos and re-sync words.

Should I fix the formatting error as well? I wanna know do others think.

Reply:
I'd prefer fixing format changes in batches rather than individually.